site/src/atomics/typography.md

```diff
@@ -25,7 +26,8 @@ The typography scale is designed for great readability across the platform. This
 | `text-decoration` | `underline`, `none`                                              | N\A        |
 | `letter-spacing`  | `wide`                                                           | N\A        |
 | `text-transform`  | `uppercase`                                                      | N\A        |
-| `text-align`      | `center`, `right`                                                | `tablet`   |
+| `text-align`      | `left`, `center`, `right`                                        | `tablet`   |
```

Comment:
Should `text-align-left` be added to the alignment section below?

Reply:
Updated